packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -438,14 +443,6 @@ const SettingsSidebarContainer = ({
                                           textClassNames="px-3 text-emphasis font-medium text-sm"
                                           disableChevron
                                         />
-                                        {HOSTED_CAL_FEATURES && (
```

Comment:
why is that removed?

Reply:
Dsync is only to be avaialble for orgs we removed this the teams section.

Reply:
but that's sso for teams?